kiwi/xml_state.py

```diff
@@ -75,6 +75,20 @@ def get_users_sections(self):
             self.xml_data.get_users()
         )
 
+    def get_description_section(self):
+        """
+        The description section of this appliance
+
+        :return: list of <description> section reference(s)
+
+        :rtype: list
+        """
+        assert len(self.xml_data.get_description()) == 1, \
+            "Only one description section must exist"
```

Comment:
This is something you don't have to do in code. The schema specifies that only one description section is allowed. Thus this would fail at the validation level and your code path would be never reached